code/_onclick/observer.dm

```diff
@@ -14,7 +14,7 @@
 
 	// Otherwise jump
 	else if(A.loc)
-		loc = get_turf(A)
+		Move(get_turf(A.loc))

```

Comment:
But they already Move (and deal with crossed) in their actual movement, this is just changing the doubleclick teleport